nall/nall/decode/cue.hpp

```diff
@@ -139,7 +140,8 @@ inline auto CUE::loadTrack(std::vector<string>& lines, u32& offset) -> Track {
   Track track;
 
   lines[offset].itrimLeft("TRACK ", 1L).strip();
-  track.type = lines[offset].split(" ").last().strip().downcase();
+  auto parts2 = nall::split(lines[offset], " ");
+  track.type = parts2.empty() ? string{} : parts2.back().strip().downcase();
```

Comment:
Suffixed variable name looks unnecessary.
```suggestion
  auto parts = nall::split(lines[offset], " ");
  track.type = parts.empty() ? string{} : parts.back().strip().downcase();
```